Uebersicht.xcodeproj/project.pbxproj

```diff
@@ -555,7 +555,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export PATH=$PATH:/usr/local/bin\ncd server && npm run-script release\n";
+			shellScript = "export PATH=$PATH:/usr/local/bin:/opt/homebrew/bin\ncd server && npm run-script release\n";
```

Comment:
homebrew changed default path on arm64 - possibly cause at least for a while people were installing both x86_64 bins to rosetta and native ones - idk, there's probs a homebrew issue about it

Reply:
we probably here want to be in safe side and do `PATH=/opt/homebrew/bin:/usr/local/bin:${PATH}`
first we try arm64 path (we assume that  if it exists we want to use it), then x86_64 one, and finally we consume whatever $PATH user has. this makes things IMHO a bit more predictable and deterministic.